### DISCUSSION TITLE
Issue with Uploading Proof of Academic Status

### DISCUSSION THREAD

**Discussion by successdang99** [community/community#123748] [Copilot Conversations]
<img width="912" alt="a" src="https://github.com/community/community/assets/100022278/bfe37666-5933-4a3a-a7ba-a218f047637e">

After clicking "Process my application" when uploading proof of academic status, there are no notifications or error messages, leaving the status of the application unclear.

**Comment by successdang99** [comment]
![IMG_6575](https://github.com/community/community/assets/100022278/f5100d75-f862-4da0-a0a6-45e7f70e8db2)

Here is my academic status as a student.

**Comment by rameshgyawali127** [comment]
#Response 1
